consensus/common/src/main/java/org/hyperledger/besu/consensus/common/PoAMetricServiceImpl.java

```diff
@@ -41,4 +47,11 @@ public PoAMetricServiceImpl(final BlockInterface blockInterface, final Blockchai
   public Address getProposerOfBlock(final BlockHeader header) {
     return this.blockInterface.getProposerOfBlock(header);
   }
+
+  @Override
+  public boolean isThisNodesAddress(final Address address) {
```

Comment:
I'm not huge on this function name.  Suggestions?

Reply:
Yeah we shouldn't add this method to the public API.  It should be a method that gets the current address which can then be compared.  That could be `getLocalNodeAddress` somewhere more generic than this, but I actually like the idea of designing the API to handle the node address and the address used as a validator to potentially be different in the future.  So maybe `getLocalSignerAddress`?